tests/test_hf_mpt_gen.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from composer.core.precision import get_precision_context
+from composer.utils import dist, get_device, reproducibility
+from omegaconf import OmegaConf as om
+
+from llmfoundry import COMPOSER_MODEL_REGISTRY
+from llmfoundry.utils import build_tokenizer
+
+
+@pytest.mark.gpu
```

Comment:
we want this to run in the cpu image, so i'd remove this mark and then just skip if device == 'gpu' and cuda not available.

Reply:
added
```
def test_init_hfhub_mpt_cpu():
    test_init_hfhub_mpt(device='cpu', attn_impl='torch')
```

Reply:
<img width="729" alt="Screenshot 2023-05-23 at 2 33 52 PM" src="https://github.com/mosaicml/llm-foundry/assets/6439018/2bc698a1-0325-499b-8ad6-5c1dd687fccc">

I think the bigger issue is that none of our tests run on the cpu img

I just emulated what was already there...